apis/node/interface.go

```diff
@@ -23,7 +23,7 @@ type Interface interface {
 
 	// SendPreCommitSector publishes the miner's pre-commitment of a sector to a
 	// particular chain and returns the identity of the corresponding message.
-	SendPreCommitSector(ctx context.Context, sectorNum abi.SectorNumber, commR []byte, ticket SealTicket, pieces ...PieceWithDealInfo) (cid.Cid, error)
+	SendPreCommitSector(ctx context.Context, sectorNum abi.SectorNumber, sealedCID cid.Cid, sealEpoch, expiration abi.ChainEpoch, pieces ...PieceWithDealInfo) (cid.Cid, error)
```

Comment:
The change of types and expiration make sense. Why change ticket to sealEpoch? Was the node only using the height from the ticket previously? Are we letting the node compute the ticket?

Reply:
The only part of the ticket which was being used was the epoch from which the ticket was plucked (used as "seal epoch"). So can we instead just pass the epoch.